hw4/hw4/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace дз4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine($"Результат вычисления формулы №24 - {Math.Round((Formula24(5) + Formula24(7) + Formula24(11)) / Formula24(13), 3)}");
```

Comment:
В данной ситуации лучше не округлять, а форматировать
```suggestion
            Console.WriteLine($"Результат вычисления формулы №24 - {(Formula24(5) + Formula24(7) + Formula24(11)) / Formula24(13), F3}");
```